src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace System.Net
 {
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, PublicKeyToken=b77a5c561934e089")]
```

Comment:
Oh it is internal on .NET Framework so yeah you need to the manual shim for this one.